### DISCUSSION TITLE
v3.14.0

### DISCUSSION THREAD

**Discussion by sojan-official** [chatwoot/chatwoot#10307] [Releases & Announcements]
## Changelog

- Vite + vue 3  : Upgrades to our frontend stack and build pipelines
- Auto populate country code in phone number field based on browser timezone
- Numerous bug fixes and improvements

### Notes: 

This update brings a significant upgrade to the Chatwoot frontend stack, paving the way for faster asset builds and smoother development of upcoming features. If you’ve made customizations to the Chatwoot code, please take note, as there are several core changes throughout. We recommend reviewing the updates to ensure everything works seamlessly with your setup.

> If you are upgrading a heroku installation, please make sure to add the nodejs buildpack before upgrade - ref: https://github.com/chatwoot/chatwoot/blob/c796832a58cc9bd045893af2756dd7bc71e91195/app.json#L61

<hr /><em>This discussion was created from the release <a href='https://github.com/chatwoot/chatwoot/releases/tag/v3.14.0'>v3.14.0</a>.</em>

**Comment by vishnu-narayanan** [comment]
For `cwctl`/Linux VM deployment, make sure your `cwctl` version is latest. The latest version of cwctl is `3.0.0`.

If you are on cwctl 2.6.0 or higher,  you will be upgraded automatically. If you are on an older version, please upgrade `cwctl` first manually and then do `cwctl --upgrade`.

ref: https://www.chatwoot.com/docs/self-hosted/deployment/chatwoot-ctl#install-or-upgrade-chatwoot-ctl

**Comment by sanchez777** [comment]
I would like to please ask that the situation for permissions be considered of great priority. I have written a [commentary](https://github.com/orgs/chatwoot/discussions/10129#discussioncomment-10893179) on the v3.13 discussion topic about basic permissions for non paid use. I believe this is a great software with great range of use, and such feature would greatly strenghten it.

**Reply by sojan-official** [reply]
@sanchez777 the permissions feature was released as an alpha to explore early feedback. As @pranavrajs  stated here : https://github.com/orgs/chatwoot/discussions/10129#discussioncomment-10781197 we will be iterating over it based on the comments from the community. we will share further updates soon.

**Comment by Lucas-sales40** [comment]
This version 3.14.0 has an error and does not load the login screen after registering for the first installation.

app/login doesn't load, just white screen

**Reply by vishnu-narayanan** [reply]
Fixed in 3.14.1

**Comment by willihangga** [comment]
related issues:

- https://github.com/chatwoot/chatwoot/issues/10319
- https://github.com/chatwoot/chatwoot/issues/10309
